Automattic/wp-calypso
# Jetpack Connect

Using a proper Redux flow for authorizing a site. cc: @johnHackworth 

To test:

Either of the authorization flows should work.

1. Run this branch at calypso.localhost:3000
- go to /jetpack/connect, and attempt to connect an unconnected Jetpack site that is [running this branch](https://github.com/Automattic/jetpack/pull/3364)
- or start on an unconnect Jetpack site [running this branch](https://github.com/Automattic/jetpack/pull/3364), and click 'Connect'
